test/integration/workarea/api/admin/payment_transactions_integration_test.decorator

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+module Workarea
+  module Api
+    module Admin
+      decorate PaymentTransactionsIntegrationTest, with: :kount do
```

Comment:
This should probably get wrapped in a check to ensure that the API plugin is installed.